compose/integrations/composable-test-cases/gradle.properties

```diff
@@ -4,7 +4,7 @@ kotlin.native.enableDependencyPropagation=false
 android.useAndroidX=true
 kotlin.version=1.8.20
 agp.version=7.3.0
-compose.version=1.5.0-dev1063
```

Comment:
Why it was dev version? The required change is released in 1.4.1 now?

Reply:
This was changed in https://github.com/JetBrains/compose-multiplatform/pull/3246.

@eymar, should we remove `compose/integrations` from `tools/replaceVersion.sh`?

The alternative is to have 1.4.1 here, and if we need to test other versions, pass them via `-Pcompose.version=`

Reply:
The tests cases were failing when running with kotlin 1.9.0-Beta. 
The required change is https://github.com/JetBrains/compose-multiplatform/pull/3170 and it should be in 1.4.1

Reply:
I checked the release/1.4.1 branch, and #3170 is not there yet.
If we want users to be able to try kotlin 1.9.0-RC with 1.4.1, it's better to cherry pick that change.

Reply:
If we need to test Kotlin 1.9.0, we can:
1. pass `-Pcompose.version=1.5.0-dev1063` every time and have `1.4.1` in the default configuration.
2. or have this folder out of the alignment with the latest stable release and remove it from `tools/replaceVersion.sh`.

What option is more convinient?

I guess, if this folder is only for our internal purposes, we can just remove it from `tools/replaceVersion.sh`

Reply:
> if this folder is only for our internal purposes

that's true.
It's rather an exception that it was needed to use a dev version here. 
Let's keep `1.5.0-dev1063` for now, and remove the folder from `tools/replaceVersion.sh`.

Reply:
Done, reverted this change, and removed folders from the script